<launch>
  <node pkg="riptide_controllers" type="calibrate.py" name="calibrate" output="screen">
    <param name="properties_file" type="string" value="$(find riptide_controllers)/cfg/vehicle_properties.yaml" />
  </node>
  <node pkg="riptide_controllers" type="goToDepth.py" name="go_to_depth" output="screen" />
  <node pkg="riptide_controllers" type="goToRoll.py" name="go_to_roll" output="screen" />
  <node pkg="riptide_controllers" type="goToPitch.py" name="go_to_pitch" output="screen" />
  <node pkg="riptide_controllers" type="goToYaw.py" name="go_to_yaw" output="screen" />
  <node pkg="riptide_controllers" type="sickTricks.py" name="sick_tricks" output="screen" />
</launch>
